accurest-core/src/test/groovy/io/codearte/accurest/dsl/WiremockGroovyDslSpec.groovy

```diff
@@ -198,6 +198,314 @@ class WiremockGroovyDslSpec extends WiremockSpec {
 			stubMappingIsValidWiremockStub(wiremockStub)
 	}
 
+	def 'should use equalToJson when body match is defined as map'() {
+		given:
+			GroovyDsl groovyDsl = GroovyDsl.make {
+				request {
+					method('GET')
+					url $(client(~/\/[0-9]{2}/), server('/12'))
+					body(
+							id: value(
+									client('123'),
+									server({ regex('[0-9]+') })
+							),
+							surname: $(
+									client('Kowalsky'),
+									server('Lewandowski')
+							),
+							name: 'Jan',
+							created: $(client('2014-02-02 12:23:43'), server({ currentDate(it) }))
+					)
+				}
+				response {
+					status 200
+				}
+			}
+		when:
+			String wiremockStub = new WiremockStubStrategy(groovyDsl).toWiremockClientStub()
+		then:
+			new JsonSlurper().parseText(wiremockStub) == new JsonSlurper().parseText('''
+			{
+				"request": {
+					"method": "GET",
+					"urlPattern": "/[0-9]{2}",
+					"bodyPatterns": [
+						{
+							"equalToJson": "{\\"id\\":\\"123\\",\\"surname\\":\\"Kowalsky\\",\\"name\\":\\"Jan\\",\\"created\\":\\"2014-02-02 12:23:43\\"}"
+						}
+					]
+				},
+				"response": {
+					"status": 200,
+				}
+			}
+			''')
+		and:
+		stubMappingIsValidWiremockStub(wiremockStub)
+	}
+
+	def 'should use equalToJson when content type ends with json'() {
```

Comment:
Now it matches if a content type header ends with json, it can be easily changed :p